src/aspire/volume/volume.py

```diff
@@ -350,16 +400,21 @@ def from_vec(cls, vec):
 
         return cls(data)
 
-    def transpose(self):
+    def transpose(self, symmetry=None):
```

Comment:
What's the use case for supplying a `symmetry` argument here? Also, it should probably be documented.

Reply:
The purpose of this is to ensure that the warning emitted by `self._result_symmetry()` happens at the same stack level whether a user calls `vol.T` or `vol.transpose()`. This way the traceback points to location in the code where `T` or `transpose` was called.

Seeing as I don't really want a user setting this parameter I should probably just use a private flag and remove this argument altogether. I'll fix this up.

Reply:
Handling this with a private flag now. Let me know what you think.